example/lib/main.dart

```diff
@@ -172,50 +172,87 @@ class _MyAppState extends State<MyApp> {
     Workmanager().cancelByUniqueName('1');
   }
 
+  Future<void> _getInstalledWidgets() async {
+    try {
+      final widgets = await HomeWidget.getInstalledWidgets();
+      if (!mounted) return;
+      await showDialog(
+        context: context,
+        builder: (buildContext) => AlertDialog(
+          title: const Text('Installed Widgets'),
+          content: Column(
+            mainAxisSize: MainAxisSize.min,
+            children: [
+              Text('Number of widgets: ${widgets.length}'),
+              const Divider(),
+              for (final widget in widgets)
+                Text(
+                  '- iOS Family: ${widget.family}, '
+                  'iOS Kind: ${widget.kind}, '
+                  'Android Widget id: ${widget.widgetId}, '
+                  'Android Class Name: ${widget.androidClassName}, '
+                  'Android Label: ${widget.label}',
```

Comment:
Maybe wrap the according lines with a `Platform` check to keep the info a bit smaller

Reply:
[fe8af81](https://github.com/ABausG/home_widget/pull/233/commits/fe8af81aec8b1e9436ee5c37fb5a595ba0bc47b5)